README.md

```diff
@@ -1,161 +1,720 @@
-# PomoDB v0.1.0 (Draft)
+# The Postmodern Database (PomoDB) Specification v0.2.0
 
 ## Editors
 
-* [Quinn Wilton](https://github.com/QuinnWilton), [Fission](https://fission.codes)
-* [Brooklyn Zelenka](https://github.com/expede), [Fission](https://fission.codes)
+- [Quinn Wilton], [Fission Codes]
+- [Brooklyn Zelenka], [Fission Codes]
 
 ## Authors
 
-* [Quinn Wilton](https://github.com/QuinnWilton), [Fission](https://fission.codes)
-* [Brooklyn Zelenka](https://github.com/expede), [Fission](https://fission.codes)
+- [Quinn Wilton], [Fission Codes]
+- [Brooklyn Zelenka], [Fission Codes]
 
-## Specs
+## Dependencies
 
-* [PomoFlow](pomo_db/pomo_flow.md)
-* [PomoLogic](pomo_db/pomo_logic.md)
-* [PomoRA](pomo_db/pomo_ra.md)
+- [PomoLogic]
+- [PomoRA]
+- [PomoStore]
+- [WebNative File System]
 
 ## Appendices
 
-* [Research](RESEARCH.md)
-
-## Links
-
-* [Fission Reactor: Dialog First Look](https://fission.codes/blog/fission-reactor-dialog-first-look/)
+- [Research]
 
 # Language
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
-
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119].
 
 # Abstract
 
-PomoDB is a content-addressable database designed for use in far-edge deployments, such as on IoT or consumer devices. It targets this use case by enabling peers to synchronize heterogenous sets of end-to-end encrypted data in an eventually consistent manner.
+PomoDB is a [content-addressable][content addressing] database. It enables far-edge deployments (such as IoT and consumer devices) by synchronizing heterogenous sets of end-to-end encrypted data between peers in an eventually consistent manner.
 
 # 1. Introduction
 
+> A concept is a brick. It can be used to build a courthouse of reason. Or it can be thrown through the window.
+>
+> ― [Deleuze] & [Guattari], [A Thousand Plateaus: Capitalism and Schizophrenia] 
+
 ## 1.1 Motivation
 
-While existing databases increasingly treat network partitions as unavoidable disturbances of a network, many mitigate these disturbances under a model that either assumes they'll be bounded in length, or by allowing them to impact the availability of some nodes in the network.
+While existing databases increasingly treat network partitions as unavoidable disturbances to uptime, many mitigate these situations under a model that either assumes they will be bounded in length, or by allowing them to have some or all nodes cease operation. Such techniques are unsuitable for far-edge and local-first applications, where disorder is the norm, network partitions are ubiquitous and unbounded, and there's an uncompromising need for availability.
```

Comment:
> or by allowing them to have some or all nodes cease operation.

I __think__ I understand what you mean by this, but it isn't super clear to me. Are you trying to say that they deal with unbounded failure by assuming that the faulty nodes are permanently removed from operation?

Reply:
I'm basically just explaining the CAP theorem

Reply:
It was quite a run-on sentence; I broke it up:

```diff
- While existing databases increasingly treat network partitions as unavoidable
- disturbances to uptime, many mitigate these situations under a model that either
- assumes they will be bounded in length, or by allowing them to have some or all 
- nodes cease operation. Such techniques are unsuitable for far-edge and local-first
- applications, where disorder is the norm, network partitions are ubiquitous and
- unbounded, and there's an uncompromising need for availability.
+ Databases increasingly treat network partitions as unavoidable disturbances to uptime.
+ Many mitigate these situations under a model that either assumes interruptions are 
+ bounded in length (e.g. a consensus round), halting operation of disconnected nodes,
+ or accepting data loss of ocnflicting writes on reconnection with a leader node. Such
+ techniques are unsuitable for far-edge and local-first applications, where disorder is
+ the norm, network partitions are ubiquitous and unbounded, and there's an uncompromising
+ need for availability.
```